agent/agent/main.go

```diff
@@ -38,6 +38,8 @@ func main() {
 		fmt.Println("Must specify rootUrl.")
 		return
 	}
+
+	cred := common.MakeCredentialFromArgs(username, password)
```

Comment:
Where does 'username' and 'password' come from here? I don't see it being extracted from the command line?